### ISSUE TITLE
NumberInput aria-label does not match displayed text

### ISSUE THREAD

**Issue by ajlohr** [carbon-design-system/carbon#2516]
NumperInput Increment/Decrement buttons have an `aria-label` that does not contain / match the svgs displayed text resulting in a DAP violation.

## Environment

> Operating system

macOS 10.13.3 (17D47)

> Browser

Chrome Version 73.0.3683.75 (Official Build) (64-bit)

> Automated testing tool and ruleset

IBM Accessibility February 2019

> Assistive technology used to verify

DAP Dynamic assessment plugin

## Detailed description

For NumberInput, the up and down buttons need to have an aria-label that contains or matches the svg's title text. For example the NumberInput renders the below html:

```
<div class="bx--number__controls">
  <button class="bx--number__control-btn up-icon" type="button" title="Increment number" aria-label="Increment number" aria-live="polite" aria-atomic="true">
    <svg class="up-icon" fill-rule="evenodd" height="5" role="img" viewBox="0 0 10 5" width="10" aria-label="choose a number" alt="choose a number">
    <title>choose a number</title>
    <path d="M0 5L5 .002 10 5z"></path>
    </svg>
  </button>

  <button class="bx--number__control-btn down-icon" type="button" title="Decrement number" aria-label="Decrement number" aria-live="polite" aria-atomic="true">
    <svg class="down-icon" fill-rule="evenodd" height="5" role="img" viewBox="0 0 10 5" width="10" aria-label="choose a number" alt="choose a number">
      <title>choose a number</title>
      <path d="M0 0l5 4.998L10 0z"></path>
    </svg>
  </button>
</div>
```

Both the increment and decrement buttons `aria-label` need to contain or match the displayed text of `choose a number` from the svgs title.

So the buttons aria-label should look like `aria-label="choose a number"` to match the svg's title.

> What version of the Carbon Design System are you using?

```
        "carbon-components": "9.84.14",
        "carbon-components-react": "6.108.0",
        "carbon-icons": "7.0.7",
```

> What did you expect to happen?

aria-label to match svg's title text for accessibility.

> What happened instead?

the aria-labels does not contain or match the svgs title text resulting in a DAP accessibility violation

> What WCAG 2.1 checkpoint does the issue violate?

Accessible name does not contain the visible text label
IBM 2.5.3 Label in Name

## Steps to reproduce the issue

1. Render NumberInput and run DAP against it
